Cargo.toml

```diff
@@ -1,39 +1,47 @@
 [package]
 name = "project_init"
-version = "3.1.23"
-authors = ["Vanessa McHale <vamchale@gmail.com>"]
+version = "4.0.0"
```

Comment:
Preemptively changed the version, though more breaking changes are to be expected (especially if we move the template engine from mustache to tera)